src/anemoi/inference/checkpoint/metadata/__init__.py

```diff
@@ -115,6 +115,22 @@ def hour_steps(self):
 
         return int(n[:-1]) * {"h": 1, "d": 24}[n[-1]]
 
+    @cached_property
+    def max_refinement_level_c(self):
```

Comment:
Where is this used? Will this come in another PR?

Reply:
I added the code that uses this method